tfhe/src/high_level_api/tests/test_fhe_number_ops.rs

```diff
@@ -1,207 +0,0 @@
-#![cfg(any(feature = "integer", feature = "shortint"))]
```

Comment:
Do these tests not make sense anymore without shortints?

Reply:
Only shortints were really tested in this, there was just a tiny integer test, but not particularly worth